pkg/generator/templates/cloud-init.gardenlinux.template

```diff
@@ -6,6 +6,12 @@ cat << EOF | base64 -d > '{{ .Path }}'
 EOF
 {{- end -}}
 
+{{- define "put-content-unencoded" -}}
```

Comment:
```suggestion
{{- define "put-content-decoded" -}}
```

or

```suggestion
{{- define "put-content-raw" -}}
```

Or even  
```suggestion
{{- define "put-content" -}}
```

and rename the  the current `put-content` to `put-content-encoded`.

Reply:
Default behaviour is to place it encoded. Special case is `raw`. So I now use `put-content` for the default case and `put-content-raw` for the special case. Thanks for the suggestion.